superset/utils/cache.py

```diff
@@ -14,14 +14,14 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-from typing import Callable, Optional
+from typing import Any, Callable, Optional
 
 from flask import request
 
 from superset.extensions import cache_manager
 
 
-def view_cache_key(*_, **__) -> str:
+def view_cache_key(*args: Any, **kwargs: Any) -> str:  # pylint: disable=unused-argument
```

Comment:
I personally prefer `*_: Any, **__: Any` if it allows us to avoid disabling linting for a row.

Reply:
@villebro I think the issue with using `_` etc. is it's really not clear to the reader what these arguments are, i.e., when I was looking at typing [this](https://github.com/apache/incubator-superset/blob/master/superset/views/database/views.py#L48) method is wasn't clear what `_` represented, and although the argument is currently unused, there may be value is using it in the future (it was provided for some purpose). I needed to add some debugging statements to determine the argument type.

Reply:
Fair point about `_` being ambiguous in this context, i.e. either signalling something that's unused or a `flask_babel` function call. I'm fine with this.